llama/model.py

```diff
@@ -194,23 +210,46 @@ def __init__(
 
         init_method = lambda x: x
 
-        self.w1 = ColumnParallelLinear(
-            dim, hidden_dim, bias=False, gather_output=False, init_method=init_method, world_size=world_size, rank=rank, groups=groups, quant=quant
-        )
-        self.w2 = RowParallelLinear(
-            hidden_dim, dim, bias=False, input_is_parallel=True, init_method=init_method, world_size=world_size, rank=rank, groups=groups, quant=quant
-        )
-        self.w3 = ColumnParallelLinear(
-            dim, hidden_dim, bias=False, gather_output=False, init_method=init_method, world_size=world_size, rank=rank, groups=groups, quant=quant
-        )
+        self.w1 = ColumnParallelLinear(dim,
```

Comment:
did you use a different formater? A lot of code changes to python are just format changes.

Reply:
I just use yapf...